### PR TITLE
GameDB: WRC avec sebastien loeb performance fixes

### PR DESCRIPTION
**Pull request by atomic83GitHub**
This PR add a modified patch to "WRC avec sebastien loeb", this fixes a very bad performance problem with a delay slot instruction which for an unknow reason (Timing?) tanks the performances when using physics in the playground mode.

With the delay slot instruction:

Capture

Without the delay slot instruction:

Capture2

### CODE REVIEW

**Comment by lightningterror** [comment]
Curious, how does the game behave on older builds before the delay branch pr was merged with that patch line commented out.

**Comment by atomic83GitHub** [comment]
It is exactly the same with the older build, if this address is commented the game will have the same slowdowns.

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -2281,6 +2281,12 @@ Serial = SCES-53680
 Name   = WRC avec Sebastien Loeb
 Region = PAL-M5
 XgKickHack = 1 // SPS.
+[patches = 79BAD675]
+	author=By Prafull and atomic83github
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: It will be better to have Prafull as author only since there aren't any actual modifications made to the patch.

**Comment by lightningterror** [comment]
> It is exactly the same with the older build, if this address is commented the game will have the same slowdowns.

Thanks, just the info I needed to make sure the branch delay pr didn't cause any new issues.

**Comment by aphelionosu** [comment]
This works well on WRC: Rally Evolved too. Big thanks

**Pull request merged**